lib/saputils.pm

```diff
@@ -84,7 +89,7 @@ sub calculate_hana_topology {
 }
 
 =head2 check_hana_topology
-    check_hana_topology([input => $saphanasr_showAttr_format_script_output]);
+    check_hana_topology(input => calculate_hana_topology($saphanasr_showAttr_format_script_output) [, online_str => '12345678']]);
```

Comment:
Example is wrong, `online_str` is not a valid argument. Name is `node_state_match`